xarray/core/ops.py

```diff
@@ -17,6 +18,10 @@
 
 try:
     import bottleneck as bn
+    if StrictVersion(bn.__version__) < StrictVersion('1.0'):
+        warnings.warn('xarray requires bottleneck version of 1.0 or greater.'
```

Comment:
Users find it really annoying to see messages just when importing a package, when they may not even be using the related functionality. Let's only issue the warning when doing rolling aggregations.